homeassistant/components/awair/const.py

```diff
@@ -95,6 +95,7 @@ class AwairSensorEntityDescription(SensorEntityDescription, AwairRequiredKeysMix
     ),
     AwairSensorEntityDescription(
         key=API_VOC,
+        device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
         icon="mdi:molecule",
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
```

Comment:
The `VOLATILE_ORGANIC_COMPOUNDS` device class cannot have a unit of measurement.

../Frenck

Reply:
Hi, thanks for the hint.
So... "CONCENTRATION_PARTS_PER_BILLION" should probably one of these then?

DeviceClass | Unit| Description
-- | -- | --
SensorDeviceClass.CARBON_DIOXIDE | ppm | Concentration of carbon dioxide.
-- | -- | --
SensorDeviceClass.CARBON_MONOXIDE | ppm | Concentration of carbon monoxide.

or the ppm should be extended to ppb...
AND ppm as well as ppb are valid unit of measurements for volatile organic compounds:
https://www.pressac.com/insights/what-are-volatile-organic-compounds-vocs-and-why-should-you-be-monitoring-them/#:~:text=Because%20there%20are%20so%20many,parts%20per%20billion%20(ppb)).

![image](https://user-images.githubusercontent.com/26391061/216560237-24558719-8ac6-4ed8-9dfb-6a20e415479b.png)

Reply:
> So... "CONCENTRATION_PARTS_PER_BILLION" should probably one of these then?

CO & CO2 are not the same as VOC.

We currently do not have a device class available for VOC sensors using the `ppm` or `ppb` units for their measurement. In those cases, we don't set a device class.

>  ppm as well as ppb are valid unit of measurements for volatile organic compounds:

Sure, but not for Home Assistant.

../Frenck